### ISSUE TITLE
[Feature Request]Multiple accounts

### ISSUE THREAD

**Issue by arbyruns** [daneden/Twift#51]
I've been playing around with the ability to add multiple accounts, but I'm not seeing that as a possibility. Am I missing something?

**Comment by ramzesenok** [comment]
One `Twift` client should be responsible for one user. If you want to support multiple users you can replace old `Twift` client bound to `user1` with a new client bound to `user2`

**Comment by arbyruns** [comment]
@ramzesenok thank you! I'll give this a try.

**Comment by arbyruns** [comment]
@ramzesenok I went ahead and gave this a shot and the problem I'm running into is that when `Twift.Authentication()` method is called the current account is still logged in and no way to log that account out.

**Comment by ramzesenok** [comment]
you mean in the browser that is opened to confirm login?

**Comment by ramzesenok** [comment]
@arbyruns you mean in the browser that is opened to confirm login?

**Comment by arbyruns** [comment]
@ramzesenok correct. I think it has to do with ASWebAuthenticationSession. If launch safari Twitter is logged in. If I log myself out the new session does not have me logged in. Does that make sense?

**Comment by ramzesenok** [comment]
@arbyruns totally, this is how it works and it has nothing to do with Twift. If you want to log in a different user you first have to log out from the current user in browser

**Comment by arbyruns** [comment]
@ramzesenok I wonder if adding support `prefersEphemeralWebBrowserSession` would help with this? My understanding is that the session in Safari does not need to remain logged in. However, the other challenge I'm finding with adding support for `prefersEphemeralWebBrowserSession` is that I can't ever get to authorization pages on Twitter.

**Comment by ramzesenok** [comment]
@arbyruns I believe it shouldn't log out cause the same account will be used when you use Twitter's web version
